src/components/Header/Header.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { ReactComponent as Logo } from "../../assets/home.svg";
+import styled from "styled-components";
+
+const StyledMenu = styled.div`
+  width: 75%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+
+  .option {
+    padding: 10px 15px;
+    cursor: pointer;
+    text-decoration: none;
+    color: black;
+  }
+
+  .logo-container {
```

Comment:
nepoužívej classy, radši to přepiš do styled component a použij na reference elementů js pointery